### PR TITLE
Test range_variant bucketing parity using anonymized experiment data

### PR DESCRIPTION
**Pull request by mrlevitas**
I used the script below to anonymize & future-proof the original zk experiment config (swap experiment & variant names w/ generic strings, set `start_ts` to 1970, and set `stop_ts` to 5000 AD) used for parity testing range_variants vs original sv/mv experiments:
```
ogs = {original_experiment_config ...}
sanitized_ogs = {f"exp_{idx}": {"id": val['id'], "name": f"exp_{idx}", "enabled": val['enabled'], "version": val['version'], "type": val['type'], "start_ts": 37173982, "stop_ts": 95623221982, "experiment": {"variants": [{"size": variant['size'], "name": variant['name'] if (variant['name'] == 'control_1' or variant['name'] == 'control_2') else f"variant_{index}"} for index, variant in enumerate(val["experiment"]["variants"])], "experiment_version": val["experiment"]['experiment_version'], "shuffle_version": val["experiment"]['shuffle_version'], "bucket_val": val["experiment"]['bucket_val'], "log_bucketing": val["experiment"]['log_bucketing']}} for idx, (_, val) in enumerate(x.items())}
```
(2 ads budget split experiments were manually removed from the config due to having > 3 decimal places in their "size" field)
and then running `sanitized_ogs` through the [ruby range_variant experiment converter script](https://github.snooguts.net/reddit/reddit-service-experiment-config/blob/main/scripts/exp_convert.rb) to create `range_variant_zk_config.json`.

The parity test runs all anonymized experiments from `original_zk_config.json` and compares them to their respective range_variant configs by getting a variant from each experiment for 1000 users per config.

I've also included the "output.json" file which has all 1000 user to variant mappings per experiment in this format:
```
results[experiment_name + ':' + uuid] = variant
```
in order to be able to transfer these mappings for comparison between baseplate languages.

### CODE REVIEW

File: tests/range_variant_tests/range_variant_parity_tests.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                    print(f"uuid: {uuid}")
+                    print(f"original variant: {og_variant}")
+                    print(f"rv variant: {rv_variant}")
+                self.assertEqual(og_variant, rv_variant)
+
+                # construct experiment/uuid to variant mapping
+                # used to compare results across baseplate languages
+                # results[experiment_name + ':' + uuid] = og_variant
+
+        # with open(RESULTS_OUTPUT, "w") as f:
```
Comment (julieahnt) [inline tests/range_variant_tests/range_variant_parity_tests.py:76]: was this for debugging?

File: tests/range_variant_tests/range_variant_parity_tests.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                )
+                rv_variant = rv_experiments.variant(
+                    experiment_name, user=user, app_name="", build_number=0
+                )
+
+                if og_variant != rv_variant:
+                    print(f"\n\nexperiment: {experiment_name}")
+                    print(f"uuid: {uuid}")
+                    print(f"original variant: {og_variant}")
+                    print(f"rv variant: {rv_variant}")
```
Comment (wuyiqicc) [inline tests/range_variant_tests/range_variant_parity_tests.py:69]: can we clean up the code? If we are doing `assertEqual` we don't need to print. If we are printing out difference we don't need to do `assertEqual`. Also clean up the commented code as well.

Comment (mrlevitas) [reply tests/range_variant_tests/range_variant_parity_tests.py:69]: Yeah, this was more for convenience (printing failing cases), the assert will make the script fail but this provides info on what failed.
Happy to take it out, just figured it'd be a nice error message if CI fails.

File: tests/range_variant_tests/range_variant_parity_tests.py (hunk 1 above)
Comment (mrlevitas) [reply tests/range_variant_tests/range_variant_parity_tests.py:76]: This was used to create the `tests/range_variant_tests/data/output.json` file, which has all 1000 user to variant mappings per experiment, in order to be able to transfer these mappings for comparison/tests in other baseplate languages.

I left these lines in to have a way to re-generate the output.json, but also happy to take them out. 
cc @wuyiqicc

**Review by wuyiqicc — APPROVED** [review]

**Pull request merged**